src/clients/deno/deno.json

```diff
@@ -2,7 +2,7 @@
   "name": "@justbe/webview",
   "exports": "./main.ts",
   "license": "MIT",
-  "version": "1.0.1-rc.1",
+  "version": "1.0.1-rc.2",
```

Comment:
Noticed I screwed this version up last time 😓. Instead of 1.0.1-rc.1 it should've been 1.0.0-rc.2